src/dms/frontend/EdFi.DataManagementService.Frontend.AspNetCore/Modules/TokenEndpointModule.cs

```diff
@@ -14,11 +17,28 @@ public void MapEndpoints(IEndpointRouteBuilder endpoints)
         endpoints.MapPost("/oauth/token", GenerateToken);
     }
 
-    internal static async Task GenerateToken(HttpContext httpContext)
+    internal static async Task GenerateToken(HttpContext httpContext, IOptions<AppSettings> appSettings, IOAuthManager oAuthManager, ILogger<TokenEndpointModule> logger, IHttpClientFactory httpClientFactory)
```

Comment:
Go ahead and install `CSharpier` and set it to auto format on save. It's very handy. The length of this method signature without wrapping clued me in that you don't have it installed.